Airport Ticket Booking System/BookingSystemUtils/InputValidation.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public class InputValidation : IInputValidation
```

Comment:
This can be  singleton, no need to create an instance from it to invoke its methods, since this class doesn't have any local state/props to maintain.